### DISCUSSION TITLE
How to display if I have much more than 2500 markdown files

### DISCUSSION THREAD

**Discussion by realliyifei** [GitbookIO/community#483] [1. How to]
I have a huge project with 4k+ markdown files. The repo seems to be limited to 2500 files. Is there any workaround?

**Comment by aikoii** [comment]
Hey @realliyifei! Could you please share the link to your repo? Is it OSS?

**Reply by realliyifei** [reply]
Hi the repo is here: https://github.com/realliyifei/capri-explore (since the gitbook only accepts 2.5k, so I didn’t upload the full dataset to the repo)

**Reply by aikoii** [reply]
Thanks for sharing! 🙏
